sample/calc-ja.y

```diff
@@ -47,8 +49,8 @@ end
 
 ---- footer
 
-puts 'Ķ������� 2 �浡'
-puts 'Q �ǽ�λ���ޤ�'
```

Comment:
It's funny that after including the encoding comment, GitHub doesn't show the characters correctly any more.